smart-cards/src/main/java/gov/va/api/health/smartcards/Dstu2MetadataController.java

```diff
@@ -0,0 +1,111 @@
+package gov.va.api.health.smartcards;
+
+import static java.util.stream.Collectors.toList;
+
+import gov.va.api.health.dstu2.api.datatypes.ContactPoint;
+import gov.va.api.health.dstu2.api.elements.Reference;
+import gov.va.api.health.dstu2.api.resources.Conformance;
+import gov.va.api.health.dstu2.api.resources.Conformance.AcceptUnknown;
+import java.util.List;
+import java.util.stream.Stream;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    value = {"/dstu2/metadata"},
+    produces = {"application/json", "application/fhir+json"})
+@AllArgsConstructor(onConstructor_ = @Autowired)
+public class Dstu2MetadataController {
+
+  private static final String NAME = "API Management Platform | Smart Cards - R4";
+
+  private final BuildProperties buildProperties;
+
+  private static List<Conformance.Contact> contact() {
+    return List.of(
+        Conformance.Contact.builder()
+            .name("API Support")
+            .telecom(
+                List.of(
+                    ContactPoint.builder()
+                        .system(ContactPoint.ContactPointSystem.email)
+                        .value("api@va.gov")
+                        .build()))
+            .build());
+  }
+
+  private static List<Conformance.RestResource> resources() {
+    return Stream.of(
+            SupportedResource.builder()
+                .type("Parameters")
+                .profileUrl("https://www.hl7.org/fhir/r4/parameters.html")
+                .build())
+        .map(SupportedResource::asResource)
+        .collect(toList());
+  }
+
+  private static List<Conformance.Rest> rest() {
+    return List.of(
+        Conformance.Rest.builder().mode(Conformance.RestMode.server).resource(resources()).build());
+  }
+
+  @GetMapping
+  Conformance read() {
+    return Conformance.builder()
+        .id("smart-cards-conformance")
+        .version("1.0.2")
+        .name(NAME)
+        .publisher("Department of Veterans Affairs")
+        .contact(contact())
+        .date(buildProperties.getTime().toString())
+        .description("Read and search support for credentials of immunization by patient.")
```

Comment:
May want a more generic description here, like "Read and search support for verifiable health cards by patient."

This description would get overriden by unifier, right? If so, I think you can ignore this comment.

Reply:
ultimately it will be overridden by the unifier